terraform/deployments/tfc-configuration/chat.tf

```diff
@@ -93,3 +93,33 @@ module "chat-production" {
     module.variable-set-chat-production.id
   ]
 }
+
+module "chat-ci" {
+  source = "github.com/alphagov/terraform-govuk-tfe-workspacer"
+
+  organization        = var.organization
+  workspace_name      = "chat-ci"
+  workspace_desc      = "Creates the GitHub Actions OIDC role in govuk-test for govuk-chat-evaluation Bedrock CI tests"
+  workspace_tags      = ["chat:ci", "test", "aws"]
+  terraform_version   = var.terraform_version
+  execution_mode      = "remote"
+  working_directory   = "/terraform/deployments/chat-ci/"
+  trigger_patterns    = ["/terraform/deployments/chat-ci/**/*"]
+  global_remote_state = true
+
+  project_name = "govuk-infrastructure"
+
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "main"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
+
+  team_access = {
+    "GOV.UK Non-Production (r/o)" = "write"
```

Comment:
I don't know quite what this time access means so I could be wrong here (and maybe something to check), however my expectation is that "GOV.UK Production" is a more privileged role than "GOV.UK Non-Production" and thus any application of the latter should be accompanied by the former.

I could be very wrong though.